src/javascript/app/base/binary_loader.js

```diff
@@ -82,7 +82,8 @@ const BinaryLoader = (() => {
         not_authenticated: () => localize('This page is only available to logged out clients.'),
     };
 
-    const loadHandler = (config) => {
+    const loadHandler = (page) => {
+        const config = pages_config[page];
```

Comment:
Would it be better to create a copy here instead of referencing the pages_config object? to prevent possible side effects.